sklearn/ensemble/gradient_boosting.py

```diff
@@ -322,8 +330,10 @@ def _update_terminal_region(self, tree, terminal_regions, leaf, X, y,
         """LAD updates terminal regions to median estimates. """
         terminal_region = np.where(terminal_regions == leaf)[0]
         sample_weight = sample_weight.take(terminal_region, axis=0)
-        diff = y.take(terminal_region, axis=0) - pred.take(terminal_region, axis=0)
```

Comment:
Please please please do not litter your work with unrelated changes. This makes it so much harder to review, particularly when it's a big patch. It also introduces unnecessary merge conflicts into other PRs modifying this code. We know the repo contains non-PEP8 code. Once it's merged we leave it.

Reply:
Actually I had a chat with @amueller about this earlier and he suggested that its a good idea to fix the non-pep8 code first whenever I start modify a file.
But I see your point. I'm sorry but I can't really do anything about it now. I'll make these changes in separate PRs in future.

Reply:
should be pretty straight-forward to undo....

Reply:
> should be pretty straight-forward to undo....

you mean manually inspect all the changes and then undo? or is there a better way?
i just thought i could better use time in doing other classifiers than undoing all pep8 fixes?

Reply:
You can do ``git checkout filename master`` and use ``git gui`` or a diff tool to look at the diff and commit.

Reply:
but yeah, not that urgent.